src/components/Button/ButtonDropdown.react.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+
+import * as React from "react";
+import { Button, Dropdown } from "../";
+import { Manager, Reference } from "react-popper";
+import type { Props as ButtonProps } from "../Button/Button.react";
+import type { Props as DropdownMenuProps } from "../Dropdown/DropdownMenu.react";
+
+import type { ReferenceChildrenProps } from "react-popper";
+
+type Props = ButtonProps & {|
+  +value?: string,
+  +children?: React.Node,
```

Comment:
Should we require its children to be of type `Dropdown.Item` ?

Reply:
We could, but I do like the idea of being able to put anything into the Dropdown?